tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Service/Microsoft.Legal.MatterCenter.Repository/SharePointInfo/Search.cs

```diff
@@ -326,7 +326,7 @@ public SearchResponseVM GetPinnedData(SearchRequestVM searchRequestVM, string li
                                 {
                                     if (sortCol != "" && sortCol.ToLower().Trim() == "mattermodifieddate")
                                     {
-                                        searchResponse.MatterDataList = userpinnedMatterCollection.Values.OrderBy(x => DateTime.ParseExact(TypeHelper.GetPropertyValue(x, sortCol).ToString(), "M/d/yyyy h:mm:ss tt", null));
+                                        searchResponse.MatterDataList = userpinnedMatterCollection.Values.OrderBy(x => DateTime.Parse(TypeHelper.GetPropertyValue(x, sortCol).ToString()));
```

Comment:
The old code was checking the exact date and time with AM and PM. But when we get the data, we are not getting the AM and PM format. Modified the code a little bit to fix the issue